great_expectations/render/renderer/v3/suite_edit_notebook_renderer.py

```diff
@@ -106,9 +105,7 @@ def __init__(
     def from_data_context(data_context: DataContext):
         suite_edit_notebook_config: Optional[NotebookConfig] = None
         if data_context.notebooks and data_context.notebooks.get("suite_edit"):
-            suite_edit_notebook_config = notebookConfigSchema.load(
-                data_context.notebooks.get("suite_edit")
-            )
+            suite_edit_notebook_config = data_context.notebooks.get("suite_edit")
```

Comment:
Why was this removed?

Reply:
Object is already in config form at this point due to changes made in variables